test/usertest.spec.js

```diff
@@ -1,163 +1,162 @@
+//         .send(invalidUsernameUser);
+
+//       expect(res).to.have.status(422);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body)
+//         .to.have.property('error')
+//         .eql('userName length must be at least 4 characters long');
+//     });
+
+//     it('It Give an error on Validation of Email', async () => {
+//       const res = await chai
+//         .request(app)
+//         .post('/api/auth/register')
+//         .send(invalidEmailUser);
+
+//       expect(res).to.have.status(422);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body)
+//         .to.have.property('error')
+//         .eql(`email must be a valid email`);
+//     });
+
+//     it('It Give an error on Validation of FirstName', async () => {
+//       const res = await chai
+//         .request(app)
+//         .post('/api/auth/register')
+//         .send(invalidFirstNameUser);
+
+//       expect(res).to.have.status(422);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body)
+//         .to.have.property('error')
+//         .eql('firstName length must be at least 3 characters long');
+//     });
+
+//     it('It Give an error on Validation of LastName', async () => {
+//       const res = await chai
+//         .request(app)
+//         .post('/api/auth/register')
+//         .send(invalidLastNameUser);
+
+//       expect(res).to.have.status(422);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body)
+//         .to.have.property('error')
+//         .eql('lastName length must be at least 3 characters long');
+//     });
+
+//     it('It Should Register a User Successfully', async () => {
+//       const res = await chai
+//         .request(app)
+//         .post('/api/auth/register')
+//         .send(realUser);
+//       expect(res).to.have.status(200);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body)
+//         .to.have.property('message')
+//         .eql('User created successfully');
+//     });
+
+//     it('It Should Give an Error when user exists', async () => {
+//       const res = await chai
+//         .request(app)
+//         .post('/api/auth/register')
+//         .send(dbInitUser);
+
+//       expect(res).to.have.status(409);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body).to.have.property('error').eql('email already exists');
+//     });
+
+//     it('It Should Give an Error when an Email is Dummy', async () => {
+//       const res = await chai
+//         .request(app)
+//         .post('/api/auth/register')
+//         .send(dummyEmailUser);
+
+//       expect(res).to.have.status(400);
+//       expect(res.body).to.be.a('object');
+//       expect(res.body)
+//         .to.have.property('error')
+//         .eql(`${dummyEmailUser.email} is not a valid Email`);
+//     });
+//     after(async () => {
+//       await models.User.destroy({ where: {}, truncate: true });
+//     });
+//   });
+// });
```

Comment:
@Procech20  Why are these tests commented out ???

Reply:
Working on this ASAP, I thought I uncommented them